routing/routing.go

```diff
@@ -371,6 +373,8 @@ func (r *Routing) startReceivingUpdates(o Options) {
 					}
 				}
 				r.log.Info("route settings applied")
+				r.Metrics.UpdateGauge("routes-total", float64(len(rt.validRoutes)))
+				r.Metrics.UpdateGauge("routes-last-update", float64(rt.created.Unix()))
```

Comment:
Looking at other places I think we have a kind of convention for metric key names: one or multiple parts joined with a dot.

Lets name these  `routes.total` and `routes.updated_timestamp`, somewhat similar to:
https://github.com/zalando/skipper/blob/96567ac366a3fb104cdec59409fa810de7e7d591/routesrv/polling.go#L40-L44

Reply:
makes sense, applied